variant.go

```diff
@@ -0,0 +1,30 @@
+package goshopify
+
+import "time"
+
+// Variant represents a Shopify variant
+type Variant struct {
+	ID                   int        `json:"id"`
+	ProductID            int        `json:"product_id"`
+	Title                string     `json:"title"`
+	Sku                  string     `json:"sku"`
+	Position             int        `json:"position"`
+	Grams                int        `json:"grams"`
+	InventoryPolicy      string     `json:"inventory_policy"`
+	CompareAtPrice       string     `json:"compare_at_price"`
```

Comment:
Sorry for the extra comments:

1. This is probably a decimal right? If so, you can use `*decimal.Decimal` like the order [`amount`](https://github.com/getconversio/go-shopify/blob/64825aa463f6dd5c7e81a0e15a402f807178e117/order.go#L100)
2. I noticed that you didn't include the [`price`](https://help.shopify.com/api/reference/product_variant#price-property) field anywhere. I think that would be a good addition to this resource.